python_modules/dagster/dagster/_loggers/__init__.py

```diff
@@ -111,13 +112,17 @@ def format(self, record):
     return logger_
 
 
-def default_system_loggers() -> Sequence[Tuple["LoggerDefinition", Mapping[str, object]]]:
+def default_system_loggers(
+    instance: Optional["DagsterInstance"],
```

Comment:
do we need to keep this optional ? worried about missing call sites over time. Maybe an explicit test using `instance_for_test` w/ overrides and `capfd`  ?